llvm/lib/Target/AArch64/AArch64TargetMachine.cpp

```diff
@@ -412,7 +417,8 @@ AArch64TargetMachine::getSubtargetImpl(const Function &F) const {
   StringRef FS = FSAttr.isValid() ? FSAttr.getValueAsString() : TargetFS;
   bool HasMinSize = F.hasMinSize();
 
-  bool IsStreaming = F.hasFnAttribute("aarch64_pstate_sm_enabled") ||
+  bool IsStreaming = ForceStreaming ||
+                     F.hasFnAttribute("aarch64_pstate_sm_enabled") ||
                      F.hasFnAttribute("aarch64_pstate_sm_body");
```

Comment:
Extremely petty comment but for `IsStreamingCompatible` you check the function attributes first followed by the command line option, but here the order is reversed.  My OCD doesn't like this :)

Reply:
Are you happy with me fixing up the other case in this patch as well? I figured it seems more sensible to check the `bool` first before calling some more compute-intensive function.

Reply:
Sure, fine by me.